games-emulation/retroarch/retroarch-1.7.4.recipe

```diff
@@ -6,11 +6,12 @@ shaders, netplay, rewinding, next-frame response times, and more!"
 HOMEPAGE="https://libretro.com/"
 COPYRIGHT="2010-2018 The RetroArch Team"
 LICENSE="GNU GPL v3"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="https://github.com/libretro/RetroArch/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="a60c2244609bb87cdb56dd8e1020c3be757569b5246141328804ebc5574327ea"
+CHECKSUM_SHA256="1c095076e2a9d2ec4e501176b3963ea2a831c1e5f025bb25386491272f14de40"
 SOURCE_FILENAME="retroarch-$portVersion.tar.gz"
 SOURCE_DIR="RetroArch-$portVersion"
+PATCHES="retroarch-1.7.4.patchset"
```

Comment:
You should use $portVersion istead the version number

Reply:
I considered it, however this patchset will be gone with the next release. My patch was already accepted upstream, hence there wouldn't be any real benefit. If you really think it's necessary I can still to it though.

Reply:
I think it can be a goot idea to use `$portVersion` because when `1.7.5` becomes available, it might need another patch (not related to the one you have upstreamed), so this line would already be OK.

BTW, except for some special cases, we  tend to use `$portVersion` :)